data/schema-config.yaml

```diff
@@ -412,15 +412,16 @@ protein:
         - dbxrefs[*]
     text:
       type: inverted
-      fields: name
+      fields: name, rec_full_name
```

Comment:
this is currently not supported, but my branch for Complexes has support for inverted fields with multiple fields, so we just need to run the script after merging that one first.

Reply:
Sounds good. I will wait for the merge. Thanks!